src/index.spec.ts

```diff
@@ -7,6 +7,159 @@ import {
   createProduct 
 } from './index';
 
-test('Testing time!', () => {
-  console.log('welcome');
-});
+describe('isInteger',()=>{
+  it('should return true if the value is integer',()=>{
+    const expected = true;
+    const value = 1;
+    const result = isInteger(value);
+    expect(result).toBe(expected);
```

Comment:
`expect(result).toBe(true)` it's also readable. Actually, you can look at `jest-extended `https://www.npmjs.com/package/jest-extended, it contains more assertions. And it has `.toBeTrue()` and `.toBeFalse()`